policyengine_us/parameters/gov/irs/deductions/itemized/reduction/agi_threshold.yaml

```diff
@@ -1,56 +1,155 @@
 description: IRS deducts the total itemized deductions amount for filers with adjusted gross income below this amount, based on filing status.
+
 SINGLE:
-  2009-01-01: 166_800
-  2013-01-01: 150_000
-  2014-01-01: 152_525
-  2015-01-01: 154_950
-  2016-01-01: 155_650
-  2017-01-01: 156_900
+  values:
+    2009-01-01: 166_800
+    2010-01-01: 0
+    2013-01-01: 250_000
+    2014-01-01: 254_200
+    2015-01-01: 258_250
+    2016-01-01: 259_400
+    2017-01-01: 261_500
+    2018-01-01: 0
+    # CBO forecast.
+    2026-01-01: 339_850
+    2027-01-01: 347_050
+    2028-01-01: 354_050
+    2029-01-01: 361_050
+    2030-01-01: 368_200
+    2031-01-01: 375_500
+    2032-01-01: 382_900
+    2033-01-01: 390_500
+    2034-01-01: 398_250
+    2035-01-01: 406_200
+  metadata:
+    uprating:
+      parameter: gov.irs.uprating
+      rounding:
+        type: downwards
+        interval: 50
 JOINT:
-  2009-01-01: 166_800
-  2013-01-01: 150_000
-  2014-01-01: 152_525
-  2015-01-01: 154_950
-  2016-01-01: 155_650
-  2017-01-01: 156_900
+  values:
+    2009-01-01: 250_200
+    2010-01-01: 0
+    2013-01-01: 300_000
+    2014-01-01: 305_050
+    2015-01-01: 309_900
+    2016-01-01: 311_300
+    2017-01-01: 313_800
+    2018-01-01: 0
+    2026-01-01: 414_836
```

Comment:
add values through 2035

while here could you also add uprating to applicable_amount.yaml?